mlflow/environment_variables.py

```diff
@@ -286,3 +286,22 @@ def get(self):
 #: Specifies the password used to authenticate with a tracking server.
 #: (default: ``None``)
 MLFLOW_TRACKING_PASSWORD = _EnvironmentVariable("MLFLOW_TRACKING_PASSWORD", str, None)
+
+#: Specifies and takes precedence for setting the basic/bearer auth on http requests.
+MLFLOW_TRACKING_TOKEN = _EnvironmentVariable("MLFLOW_TRACKING_TOKEN", str, None)
```

Comment:
```suggestion
#: Specifies and takes precedence for setting the basic/bearer auth on http requests.
#: (default: ``None``)
MLFLOW_TRACKING_TOKEN = _EnvironmentVariable("MLFLOW_TRACKING_TOKEN", str, None)
```

Can we add the default value?

Reply:
sure